communication/src/main/java/org/jboss/da/communcation/pnc/PNCProducer.java

```diff
@@ -0,0 +1,36 @@
+package org.jboss.da.communcation.pnc;
+
+import org.jboss.da.common.json.DAConfig;
+import org.jboss.da.common.util.Configuration;
+import org.jboss.da.common.util.ConfigurationParseException;
+import org.jboss.da.communcation.pnc.authentication.PNCAuthFilter;
+import org.jboss.resteasy.client.jaxrs.ResteasyClient;
+import org.jboss.resteasy.client.jaxrs.ResteasyClientBuilder;
+import org.jboss.resteasy.client.jaxrs.ResteasyWebTarget;
+
+import javax.enterprise.inject.Produces;
+import javax.inject.Inject;
+
+public class PNCProducer {
+    Configuration config = new Configuration();
+    /**
+     * Factory to create an instance of org.jboss.da.communcation.pnc.PNCInterface
+     *
+     * @return instance of org.jboss.da.communcation.pnc.PNCInterface
+     */
+    @Produces
+    public PNC getPNCInstance() {
+        ResteasyClient client = new ResteasyClientBuilder().build();
+
+        try (DAConfig conf = config.getConfig()) {
```

Comment:
There is no point for using try-with-resources, as DAConfig shouldn't be closable.